queue-health/queue-health-prod.yaml

```diff
@@ -23,7 +23,7 @@ spec:
           mountPath: /creds
           readOnly: true
       - name: poller
-        image: gcr.io/google-containers/queue-health-poll:v20161020
+        image: gcr.io/k8s-testimages/queue-health-poll:v20170406
```

Comment:
There is now `v20170407` which contains fixes from https://github.com/kubernetes/test-infra/pull/2422